states/DE/governor.yaml

```diff
@@ -0,0 +1,36 @@
+contact_form:
+  steps:
+    - visit: "https://governor.delaware.gov/email-governor-carney/"
+    - fill_in:
+        - name: "name"
+          selector: "#name"
+          value: $NAME_FULL
+          required: true
+        - name: "email"
+          selector: "#email"
+          value: $EMAIL
+          required: true
+        - name: "phone"
+          selector: "#phone"
+          value: $PHONE_DIGITS_ONLY
+          required: true
+        - name: "subject"
+          selector: "#subject"
+          value: $SUBJECT
+          required: true
+        - name: "description"
+          selector: "#description"
+          value: $MESSAGE
+          required: true
+        - name: "CaptchaInput"
+          selector: "#CaptchaInput"
+          captcha_selector: "#CaptchaDiv"
+          captcha_append_guidance: "Type the above number:"
```

Comment:
Lol what is this captcha!!?!??? Bizarre.

Reply:
It's actually so funny, it's literally just numbers in a div. It's not even an image. Does that...count...as a captcha?